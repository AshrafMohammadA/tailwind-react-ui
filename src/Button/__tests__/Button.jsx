import React from 'react'
import { shallow } from 'enzyme'

import { Button } from '..'

const setup = (testProps = {}) => {
  const props = Object.assign(testProps)

  const wrapper = shallow(<Button {...props}>Submit</Button>)

  return {
    props,
    wrapper,
  }
}

describe('Button', () => {
  it('renders matching snapshot', () => {
    const { wrapper } = setup({ fill: true })

    expect(wrapper).toMatchSnapshot()
  })
})
