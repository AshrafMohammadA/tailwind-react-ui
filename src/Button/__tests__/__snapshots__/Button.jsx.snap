// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders matching snapshot 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
